text/0118-search-api.md

```diff
@@ -108,156 +109,350 @@ It returns the number of documents matching the current search query for each gi
+| [`hits`](#321-hits)                                   | Array[Hit]                   | True     |
+| [`limit`](#322-limit)                                 | Integer                      | True     |
+| [`offset`](#323-offset)                               | Integer                      | True     |
+| [`nbHits`](#324-nbhits)                               | Integer                      | True     |
+| [`exhaustiveNbHits`](#325-exhaustivenbhits)           | Boolean                      | True     |
+| [`facetsDistribution`](#326-facetsdistribution)       | Object                       | False    |
+| [`exhaustiveFacetsCount`](#327-exhaustivefacetscount) | Boolean                      | False    |
+| [`processingTimeMs`](#328-processingtimems)           | Integer                      | True     |
+| [`query`](#329-query)                                 | String                       | True     |
 
-##### 1.2.2.1 `hits`
+#### 3.2.1. `hits`
 
 - Type: Array[Hit]
 - Required: True
 
-Results of the query as an array of documents.
+Results of the search query as an array of documents.
+
+> Hit object represents a matched document as a search result.
+
+> The search parameters `attributesToRetrieve` influence the returned payload for a hit. See [3.1.7. `attributesToRetrieve`](#317-attributestoretrieve) section.
 
-> The search parameters `attributesToRetrieve` influence the returned payload for a document as a search result. See 1.2.1.7 `attributesToRetrieve` section.
+A search result can contain special properties. See [3.2.1.1. `hit` Special Properties](#3211-hits-special-properties) section.
 
-> A Hit object that represents a document within the search results can host special attributes. See 1.2.2.9 `hits` special fields section.
+##### 3.2.1.1. `hit` Special Properties
 
-##### 1.2.2.2 `limit`
+| Field                                | Type        | Required |
+|--------------------------------------|-------------|----------|
+| [`_geoDistance`](#32111-geodistance) | Integer     | False    |
+| [`_formatted`](#32112-formatted)     | Object      | False    |
+| [`_matchesInfo`](#32113-matchesinfo) | Object      | False    |
+
+###### 3.2.1.1.1. `_geoDistance`
+
+- Type: Integer
+- Required: False
+
+Search queries using `_geoPoint` returns a `_geoDistance` field containing the distance in meters between the document `_geo` coordinates and the specified `_geoPoint`.
+
+> See [GeoSearch](0059-geo-search.md)
+
+###### 3.2.1.1.2. `_formatted`
+
+- Type: Object
+- Required: False
+
+`_formatted` returns highlighted and cropped attributes specified in `attributesToHighlight` and/or `attributesToCrop` of a search result.
+
+- If `attributesToHighlight` and `attributesToCrop` are not set, `_formatted` is not returned.
+- If cumulated fields in `attributesToHighlight` and `attributesToCrop` resolve to only having non-existent fields, `_formatted` is not returned.
```

Comment:
If I understand this correctly, it could be rewritten as:

```suggestion
- If none of the fields contained in `attributesToHighlight` and `attributesToCrop` are present in any of the matched documents, `_formatted` is not returned.
```